File: 0217. Contains Duplicate/Solution2.java
class Solution {
    public boolean containsDuplicate(int[] nums) {
        HashMap<Integer, Integer> map = new HashMap<>();
        for (int i = 0; i < nums.length; i++){
            if (map.containsKey(nums[i])){
                map = null;
                return(true);
            }
            map.put(nums[i], i);
        }
        map = null;
        return(false);
    }
}

File: 0027. Remove Element/Solution2.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        ans = 0
        size = len(nums)
        while ans < size:
            if nums[ans] == val:
                nums.pop(ans)
                size -= 1
            else:
                ans += 1
        return ans